File: cppjs/basics/2fibo.js
let fibo=(n)=>{
    let a=0,b=1;
    console.log(a," ",b," ");

    for(let i=3;i<=n;i++){
        let _fibo=a+b;
        a=b;
        b=_fibo;
        console.log(_fibo," ");
    }
}

console.log(fibo(9));

File: cppjs/basics/2fibo.cpp
#include<iostream>

void fibo(int n){
    int a=0,b=1;
    std::cout<<a<<" "<<b<<" ";

    for(int i=3;i<=n;i++){
        int fibo=a+b;
        a=b;
        b=fibo;
        std::cout<<fibo<<" ";
    }
    std::cout<<std::endl;
}

int main(){
    int n;
    std::cin>>n;
    fibo(n);

    return 0;
}